struct TestValue {
  1: i64 legalClicks,
  2: i64 rawImpressions,
  3: i64 lastLegalClickTime,
  4: i64 lastLegalImpressionTime,
  5: i64 socialClicks,
  6: i64 socialImpressions,
  7: double throttle,
  8: i64 pageOtherImpressions,
  9: i64 pageCanvasImpressions,
  10: i64 pageProfileImpressions,
  11: i64 pageSearchImpressions,
  12: i64 pageEventImpressions,
  13: i64 pageGroupImpressions,
  14: i64 pagePhotoImpressions,
  15: i64 pageHomeImpressions,
  16: i64 position1Impressions,
  17: i64 position2Impressions,
  18: i64 position3Impressions,
  19: i64 feedbackLegalLikes,
  20: i64 feedbackLegalDislikes,
  21: list<i64> rollingWindowLegalClicks,
  22: list<i64> rollingWindowRawImpressions,
  23: list<i64> rollingWindowFeedbackLegalLikes,
  24: list<i64> rollingWindowFeedbackLegalDislikes,
  25: i64 pageAlbumImpressions,
  26: i64 pageAdBoardImpressions,
  27: string text,
}

struct TestOtherValue {
  1: required string color,
}
